README.md

```diff
@@ -51,9 +51,9 @@ below.
 
 ```bash
 # switches to your home directory
-cd
+cd %userprofile%
 # activates the conda environment you installed
-datafest\Scripts\activate.bat
+datafest\Scripts\activate
```

Comment:
On my windows 7 box, there was no "activate" only "activate.bat". Yours didn't have the .bat?